### DISCUSSION TITLE
Is there an endpoint for cloud imagery?

### DISCUSSION THREAD

**Discussion by davidlfox** [weather-gov/api#706] [Q&A]
Does this API provide endpoints for regional cloud cover images? Context is a mobile app for an astronomy club. Lat/Long/Zoom would always be the same. Maybe grab an image every ~15 minutes and cache it so multiple users don't bombard servers. The closest I can find so far is the sky cover percentage, which isn't as helpful to active observers.

**Comment by dwhitemv25** [comment]
You're looking for infrared satellite imagery, then? The API does not provide satellite images, unless they added it recently and I didn't notice.

There are sector images at STAR ([star.nesdis.noaa.gov](https://www.star.nesdis.noaa.gov/GOES/index.php)) though the sectors are large. 

There's other sources in NOAA for those, but I'm blanking on where they are right now. Worse case, if you know where to look, you can find the raw satellite data, but then you need the tools to convert it to a common image format, reproject/warp, etc.

Some third-party providers may have map tiles or subset features you could use.
